odometry_conversion/launch/tracking_camera_sim.launch

```diff
@@ -5,5 +5,6 @@
     <param name="in_sensor_frame" value="tracking_camera_link"/>
     <param name="out_odom_frame" value="tracking_camera_odom"/>
     <param name="out_sensor_frame" value="base_link"/>
```

Comment:
It might make more sense to call this is out_base_frame instead of out_sensor_frame ?

Reply:
Yes, I agree. I think this would cause less confusion. either `out_base_frame` or `out_ref_frame` or just `out_frame`

Reply:
Yeah, I think makes sense. I will probably change it to out_base_frame, it is easier to see that we are talking about something on the base.